apps/app/src/components/AddQuestionModal/AddQuestionModal.tsx

```diff
@@ -22,36 +22,48 @@ export const AddQuestionModal = (props: ComponentProps<typeof BaseModal>) => {
 	const [content, setContent] = useState("");
 	const [isError, setIsError] = useState(false);
 
-	const { openModal, closeModal } = useUIContext();
-	const { createQuestionMutation } = useCreateQuestion();
+	const { openModal, closeModal, modalData, openedModal } = useUIContext<"AddQuestionModal">();
+	const { createQuestionMutation, patchQuestionMutation } = useQuestionMutation();
 
 	const disabled =
 		!selectData.technology ||
 		!selectData.level ||
 		content.length === 0 ||
 		createQuestionMutation.isLoading;
 
+	useEffect(() => {
+		if (modalData) {
```

Comment:
Nie powinno być przypadkiem tak?
```suggestion
		if (openedModal && modalData) {
```